src/olympia/amo/pagination.py

```diff
@@ -48,7 +48,7 @@ def page(self, number):
         bottom = (number - 1) * self.per_page
         top = bottom + self.per_page
 
-        if bottom > self.max_result_window:
+        if (number * self.per_page) > self.max_result_window:
```

Comment:
I think it would be less confusing if you simply used `top` here. Or am I missing something?

Reply:
no, I think you're not missing something. I actually did, that seems to work fine afaik.

Fixed in ca1826d